### DISCUSSION TITLE
Virtual Community Meeting?

### DISCUSSION THREAD

**Discussion by flomickl** [apache/streampipes#1033] [General]
Hi all,

I think this was a topic some time ago.
How about a virtual community meeting where we can talk and discuss future developments?
* getting to know each other
* Scrum-like discussion on which tickets to work on until the next meeting.
* Getting to know the background topics of these steps.
* Knowing how to contribute
* Good way to create a documentation from this meeting about contributions, background information and so on.
* Hot topics that someone might want to know about that are too complicated to explain via email.
* And I think a lot more....

So, beside the question "if we do this", there is also how and where?

Do we have a zoom, teams, WebEx or whatever possibility to meet in a virtual room?
How to find a suitable date.
How often would we do this meeting? Monthly... 2 month ...?
Who would prepare the agenda and collect topics.
Who is doing the documentation and "protocol"
...
And I think a lot more tasks

What do you think?

Greetings
Florian

**Comment by bossenti** [comment]
Hi Florian,

thanks for your initiative.
I really like the idea of a virtual community meeting.
From my point of view a monthly schedule would be a good fit, but maybe let's just start with a first meeting and then step by step find our rhythm and our own style of conducting etc.
With respect to the technical setup, I can offer to host a Google Meet session.

For a first meeting, I could imagine to have an introduction round (maybe even as a small game) so that we get to know each other slightly and afterwards we could do a lean coffee format, where everyone can bring up topics he/she wants to talk about. I would volunteer to organize and moderate this aspect.

Curious, what the others think  
Hope we get a meeting started soon 🤩

**Comment by tenthe** [comment]
I am completely in favor of this. Thanks a lot Florian for taking the initiative. What do you think of the last Wednesday of the month? So the first meeting on 25.01. This is just an arbitrary day to start the discussion ;) we can do it on another day if you want to.

Regarding the tool I am flexible. We can do it with Google Meet.
What time would you suggest?

**Comment by flomickl** [comment]
I think the time really depends on the spatial distribution of the people who will participate in the meeting correlated to UTC Timezone :)
Last Wednesday of the month sounds great for me.

**Comment by bossenti** [comment]
I agree with you @tenthe we just need some date to start with, so the 25th of January is fine for me.
From my German perspective something around 8 pm CET (7pm UTC) would be fine, but I'm totally fine with adjusting this in accordance with people living in other timezones.

**Reply by bossenti** [reply]
How about creating a poll which we share both here and in the mailing list where people can enter times that would work for them? 
I can create one if you are fine with that 😀

**Comment by bossenti** [comment]
I've created a quick poll and distributed it via our dev mainling list.

For those interested in attending our community meeting, please subscribe with our [dev mailing list](https://streampipes.apache.org/mailinglists.html).
We will share updates and the invitation link there.

**Reply by bossenti** [reply]
@aamirxshaikh please also consider yourself invited 🙂

**Reply by aamirxshaikh** [reply]
Sure @bossenti, would love to attend.

**Reply by bossenti** [reply]
Awesome 🚀 
 @aamirxshaikh  Here is the link to the message on our dev list: https://lists.apache.org/thread/qjmvgqz2vs60pcm4d3nvzjrzbl20hnqt

It would be great if you could subscribe to it anyway, so you don't miss any updates. 
If there are any problems in subscribing, please reach out 🙂

**Reply by aamirxshaikh** [reply]
Sure @bossenti

**Comment by flomickl** [comment]
@bossenti I didn't get the final result. Shall we meet now at 8:00 or 9:00?

**Reply by bossenti** [reply]
@flomickl 
We'll start at 8am UTC.
Please see the meeting link on the dev list: https://lists.apache.org/thread/s8gznrj15oqkznk44kongdkgov41qsgp
